code/modules/modular_computers/computers/modular_computer/interaction.dm

```diff
@@ -0,0 +1,300 @@
+/obj/item/modular_computer/proc/update_verbs()
+	verbs.Cut()
+	if(ai_slot)
+		verbs |= /obj/item/modular_computer/verb/eject_ai
+	if(portable_drive)
+		verbs |= /obj/item/modular_computer/verb/eject_usb
+	if(card_slot && card_slot.stored_card)
+		verbs |= /obj/item/modular_computer/verb/eject_id
+	if(stores_pen && istype(stored_pen))
+		verbs |= /obj/item/modular_computer/verb/remove_pen
+
+	verbs |= /obj/item/modular_computer/verb/emergency_shutdown
+
+// Forcibly shut down the device. To be used when something bugs out and the UI is nonfunctional.
+/obj/item/modular_computer/verb/emergency_shutdown()
+	set name = "Forced Shutdown"
+	set category = "Object"
+	set src in view(1)
+
+	if(usr.incapacitated() || !istype(usr, /mob/living))
```

Comment:
Second check is redundant. `/mob/incapacitated()` always return `TRUE`

Reply:
fixed